src/main/java/org/cryptomator/cryptofs/CryptoFileSystemProperties.java

```diff
@@ -44,6 +45,15 @@ public class CryptoFileSystemProperties extends AbstractMap<String, Object> {
 
 	static final int DEFAULT_MAX_CLEARTEXT_NAME_LENGTH = LongFileNameProvider.MAX_FILENAME_BUFFER_SIZE;
 
+	/**
+	 * Shortening threshold for ciphertext filenames.
+	 *
+	 * @since 2.5.0
+	 */
+	public static final String PROPERTY_SHORTENING_THRESHOLD = "shorteningThreshold";
+
+	static final int DEFAULT_SHORTENING_THRESHOLD = Constants.DEFAULT_SHORTENING_THRESHOLD;

```

Comment:
Is `Constants.DEFAULT_SHORTENING_THRESHOLD` still used in other places? Otherwise feel free to drop it.

Reply:
Done in https://github.com/cryptomator/cryptofs/pull/137/commits/96251235c52630ac5daf456db3b702a44eee99de